src/app/space/create/codebases/codebases.component.html

```diff
@@ -1,38 +1,76 @@
+  </f8-feature-toggle>
   <div class="container-fluid">
     <div class="row">
       <div class="col-md-12">
-        <pfng-list
-          [actionHeadingTemplate]="actionHeadingTemplate"
-            [actionTemplate]="actionTemplate"
-            [config]="listConfig"
-            [items]="codebases"
-          [itemHeadingTemplate]="itemHeadingTemplate"
-            [itemTemplate]="itemTemplate"
-            [expandTemplate]="expandTemplate"
-            (onActionSelect)="openAddCodebasePanel()">
-        <ng-template #itemHeadingTemplate let-item="item">
-          <codebases-item-heading [cheState]="cheState" class="codebase-item"></codebases-item-heading>
-        </ng-template>
-          <ng-template #itemTemplate let-item="item" let-index="index">
-            <codebases-item [cheState]="cheState" [codebase]="item" [index]="index" class="codebase-item"></codebases-item>
+        <f8-feature-toggle featureName="AppLauncher">
+          <pfng-list
+            [actionHeadingTemplate]="actionHeadingTemplate"
+              [actionTemplate]="actionTemplate"
+              [config]="listConfig"
+              [items]="codebases"
+            [itemHeadingTemplate]="itemHeadingTemplate"
+              [itemTemplate]="itemTemplate"
+              [expandTemplate]="expandTemplate"
+              (onActionSelect)="openAddCodebasePanel()" default-level>
+          <ng-template #itemHeadingTemplate let-item="item">
+            <codebases-item-heading [cheState]="cheState" class="codebase-item"></codebases-item-heading>
+          </ng-template>
+            <ng-template #itemTemplate let-item="item" let-index="index">
+              <codebases-item [cheState]="cheState" [codebase]="item" [index]="index" class="codebase-item"></codebases-item>
+            </ng-template>
+          <ng-template #actionHeadingTemplate let-item="item">
+            <!-- Extra margin for kebab -->
+            <span class="margin-right-5"></span>
           </ng-template>
-        <ng-template #actionHeadingTemplate let-item="item">
-          <!-- Extra margin for kebab -->
-          <span class="margin-right-5"></span>
-        </ng-template>
-        <ng-template #actionTemplate let-item="item" let-index="index" class="codebase-item">
-            <codebases-item-actions [cheRunning]="cheState && cheState.running" [codebase]="item" [index]="index"></codebases-item-actions>
+          <ng-template #actionTemplate let-item="item" let-index="index" class="codebase-item">
+              <codebases-item-actions [cheRunning]="cheState && cheState.running" [codebase]="item" [index]="index"></codebases-item-actions>
+            </ng-template>
+            <ng-template #expandTemplate let-item="item" let-index="index">
+              <codebases-item-details [codebase]="item"></codebases-item-details>
+            </ng-template>
+          </pfng-list>
+        </f8-feature-toggle>
+        <f8-feature-toggle featureName="AppLauncher">
```

Comment:
I see pfng-list wrapped twice with the AppLauncher feature flag. Can you wrap both pfng-list components with a single feature flag and use user-level and default-level?

Reply:
Sure, Dan.